Libraries/Opc.Ua.Server/NodeManager/MasterNodeManager.cs

```diff
@@ -2710,6 +2794,11 @@ public IList<INodeManager> NodeManagers
             get { return m_nodeManagers; }
         }
 
+        /// <summary>
+        /// The namespace managers being managed
+        /// </summary>
+        internal INodeManager[][] NamespaceManagers => m_namespaceManagers;
```

Comment:
I added this internal getter to give us something to assert on in the unit test harness. 

I figured this was a decent enough compromise to the `public NodeManagers` property but am certainly open to suggestions and/or alternatives here

Reply:
looks good to me